src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.18.0
+1.11.1-20210730+git18df1e1db3
```

Comment:
Disregard "1.11.1" here, as it's related to an outstanding issue with `calc_release_version.py` ([CDRIVER-3315](https://jira.mongodb.org/browse/CDRIVER-3315)).